DOCS-#6819: Update Modin on cluster documentation (#6678)
modin-project/modin: docs/getting_started/using_modin/using_modin_cluster.rst

This configuration script starts 1 head node (m5.24xlarge) and 7 workers (m5.24xlarge),
768 total CPUs. For more information on how to launch a Ray cluster across different
cloud providers or on-premise, you can also refer to the `Ray's cluster docs`_.

.. note::

>    By default, Modin uses 60% of the system memory. It is recommended to use the same

Reviewer: ```suggestion
   By default, Modin on Ray uses 60% of the system memory. It is recommended to use the same
```
Why 60%?

Author: > Why 60%

https://github.com/modin-project/modin/blob/324099d8737ead092e4acacfe03b1caa82e01986/modin/core/execution/ray/common/utils.py#L44

Congratualions! You have successfully connected to the Ray cluster.

.. note::

>    Be careful when using the Ray client to connect to a remote cluster, most likely
>    this connection mode will not work. Known bugs:

Reviewer: ```suggestion
   Be careful when using the Ray client to connect to a remote cluster.
   This connection mode may not work. Known bugs:
```

.. _`DataFrame`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html
.. _`pandas`: https://pandas.pydata.org/pandas-docs/stable/
.. _`open an issue`: https://github.com/modin-project/modin/issues
.. _`Ray's API docs`: https://ray.readthedocs.io/en/latest/api.html
.. _`Part 1`: https://github.com/modin-project/modin/tree/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/exercise_5.ipynb
.. _`Part 2`: https://github.com/modin-project/modin/tree/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/exercise_6.ipynb
.. _`Ray's autoscaler options`: https://docs.ray.io/en/latest/cluster/vms/references/ray-cluster-configuration.html#cluster-config
.. _`Ray's cluster docs`: https://docs.ray.io/en/latest/cluster/getting-started.html
.. _`NYC Taxi dataset`: https://modin-datasets.s3.amazonaws.com/testing/yellow_tripdata_2015-01.csv

> .. _`Modin's cluster setup config`: https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/cluster/modin-cluster.yaml

Reviewer: It seems this file is stale and we should update it as part of a separate issue. Actually, we would need update cluster modes for all distr backends - Ray, Dask, MPI.

Author: > It seems this file is stale and we should update it as part of a separate issue.

Indeed. Maybe @Retribution98 can look at it? Here one will need to create a cluster in AWS.

> Actually, we would to need update cluster modes for all distr backends - Ray, Dask, MPI.

Don't get it.

Reviewer: > > It seems this file is stale and we should update it as part of a separate issue.
> 
> Indeed. Maybe @Retribution98 can look at it? Here one will need to create a cluster in AWS.

We will schedule this offline.

> > Actually, we would to need update cluster modes for all distr backends - Ray, Dask, MPI.
> 
> Don't get it.

I mean docs/examples regarding Modin on cluster.
